HomeUI/src/views/apps/Management.vue

```diff
@@ -4628,11 +4625,11 @@ export default {
         const continent = this.forbiddenGeolocations[`selectedContinent${i}`];
         const country = this.forbiddenGeolocations[`selectedCountry${i}`];
         const region = this.forbiddenGeolocations[`selectedRegion${i}`];
-        if (continent && continent !== 'NONE') {
```

Comment:
I think this was ok, on forbidden, if none is selected on continent it should say None not all. As it will not set anything if the it's on the default choice.